pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -43,14 +51,18 @@ type fatalError string
 type transientError string
 
 func (e fatalError) Error() string {
-	return fmt.Sprintf("fatal error handling DeploymentConfig: %s", string(e))
+	return fmt.Sprintf("fatal error handling deploymentConfig: %s", string(e))
 }
 func (e transientError) Error() string {
-	return "transient error handling DeploymentConfig: " + string(e)
+	return "transient error handling deploymentConfig: " + string(e)
 }
 
 // Handle processes config and creates a new deployment if necessary.
 func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig) error {
+	// Inspect a deployment configuration every time the controller reconciles it
+	if details := c.failureInspection(config); len(details) > 0 {
+		c.detailsUpdate(config, details)
```

Comment:
> Then findDetails needs to only be called once, from here (the one on L94 would not be necessary).

I was aware of it but wanted to keep population and cleaning in separate places. I guess though it doesn't really matter. Updating...